src/Game.tsx

```diff
@@ -151,6 +148,11 @@ function GameReview(props: {
       <div className="final-score-container">
         <h2 className="final-score">Score: {score}</h2>
       </div>
+      <h2 className="new-high-score">
+        {highScores.length === 0 || score >= highScores[0]
+          ? "New high score!"
+          : ``}
```

Comment:
Rather than an empty string in an h2, let's move the conditional above the `<h2>` and just not render the node at all when there's no new high score